### PR TITLE
Business: Nassim Taleb

### PR DESCRIPTION
**Pull request by michaelfolkson**

### CODE REVIEW

**Comment by j2kun** [comment]
I would prefer a link to a post that is actually about business or quantifying risk, trying to inform instead of just being gratuitously negative. It seems nothing on his medium blog is actually about his area of expertise, and he writes about things solidly outside his area of expertise... Sorry, but I'm closing this PR.

**Comment by michaelfolkson** [comment]
That's why I originally set up a category on Risk. Risk and uncertainty underlie every subject. There aren't too many polymaths around (with a solid background in math and probability) but Taleb is certainly one of them. I expect you to call out Balaji next time he Tweets on something other than technology to be consistent :)

**Comment by j2kun** [comment]
The difference is Taleb is not my employer. Nevertheless, Balaji's twitter feed would also not be appropriate for this repository. The problem with both is that they aren't writing about risk, they're writing about politics and economics (and they're neither politicians nor economists). It doesn't help that Taleb's criticisms about "Intellectual Yet Idiots" and "no skin in the game" apply equally well to himself.

My point is that I want writers who write about their area of expertise. If Taleb wrote a blog with the content of his risk certification programs (or whatever you call it, the content of his teaching and not his political pulpit) then I would put that here. I don't care what Taleb thinks about people who try to impose virtue on other people. If instead he described scientific experiments in psychology that attempted to create reproducible environments in which people expressed those behaviors (a la Stanford prison experiment, putting aside IRB/ethics issues), this would be better, and one might be inclined to consider him more of an expert. Instead he fills his blog with anecdotes, analogies, and critiques of social engagement backed up by made-up ad hoc "heuristics."

**Comment by michaelfolkson** [comment]
Fair enough. His Medium blogs certainly aren't as strong as his books and papers on risk (that in my opinion are exceptional). But I do personally like the fact that both Taleb and Balaji wander onto other topics. They are often a lot more insightful and less biased than those who are considered experts in those fields. Most history, politics, economics "experts" struggle with logic, are unable to analyze empirical evidence and do not identify the uncertainty around their conclusions, predictions. Taleb/Balaji may fall into these traps sometimes but their training means they are better equipped to challenge the opinions of these "experts" and prevailing opinion than the "experts" themselves.

**Comment by michaelfolkson** [comment]
Plus we all have to act professionally and prioritize but I hope that at 21 you are allowed to have different opinions and discuss them (when you get the chance). Employment shouldn't change (or suppress) your opinion overnight. I've never worked there but from the outside the culture of a16z (Balaji's last employer) seems to be very encouraging of debate and differences in opinion.

**Comment by sebinsua** [comment]
My two-cents: Taleb is definitely an interesting thinker, however his blogs are his most popular writing and not his best content. He seems to save this for his books, or [to only post occasional excerpts on his site](http://fooledbyrandomness.com/SITG.html).

**Comment by j2kun** [comment]
@michaelfolkson one idea which I am coming to believe more and more with regards to "external experts" is that deep experience usually trumps logic, because the "logic" often assumes to much, oversimplifies, or falsely analogizes (while still being valid logic, it's unsound). Sooner or later their attention span runs out. Meanwhile, the lifetime experts who actually make decisions have to balance the logical empirical data and the illogical (partisan feuds, human behavior, randomness) that twitter experts can freely shed. Challenges and criticism are cheap.

21 has a very open intellectual culture, and I feel free to voice my opinions. I have done my fair part correcting Balaji when he cites a study that was later shown to be not reproducible or confounded by other factors. However, I find the nuances of the system we're designing far more interesting to discuss. Otherwise it's a bunch of engineers regurgitating what they read online to each other without any knowledge.

**Comment by j2kun** [comment]
@sebinsua I haven't had time to read through all of these (I'm on vacation in Budapest spending time with family and my grandmother who just turned 91 (!!)). I will be removing the ones I don't like later. 

For Paul Graham, he at least regularly writes about startups, entrepreneurship, and how to raise money. Taleb's blog is empty of his expertise.

I also find, eg., Scott Alexander's writing to be too meandering (AI risk? You're a psychiatrist), but the quality of his posts directly about the medical system, medical research papers, and his personal experience are worth wading through the junk. He will also overuse analogies, but he will at least admit when it's a stretch.

So in short, Taleb doesn't meet my personal BS threshold.

**Comment by michaelfolkson** [comment]
Cool. Enjoy time with family and happy new year!
